tests/unit/large_inserts_bugs_stress_test.c

```diff
@@ -0,0 +1,661 @@
+// Copyright 2022 VMware, Inc.
```

Comment:
Haven't we rejected adding this test in the past?  I don't see a lot of shmem-specific stuff to it.

Reply:
@rtjohnso - Yes, we discussed this previously. Over the 3-part stabilization this test has added value in finding bugs that are otherwise not found by existing stress / perf tests.

Even in this round of stabilization, if I enable the calls to this test from `test.sh`, some cases will randomly fail. We surface trunk bugs and so on.

Stabilizing all those as part of this feature dev integration is way too much than I can take on now.

For now, I have left this file in the change set, and it will still be compiled and built.

As we move thru part-2 and part-3, you will see some merges for this file. 

I think we should revisit and reconsider whether to bring this test suite online after part-3 is completely reviewed and integrated.

Let's discuss ... in our closure meeting.